### DISCUSSION TITLE
Translucent background in playback options

### DISCUSSION THREAD

**Discussion by Parsa-GP** [ppy/osu#29588] [Ideas]
The options dialog in the playback blocks the actual playback view and skin elements, so it would be nice to see through it.
In the attached image, you can see the dialog blocks the key counter and i have to move my mouse out of dialog to see the key.

An option to move the options would be better imo.

It's the same look on Audio and Visual settings.
![osu_2024-08-24_17-25-54](https://github.com/user-attachments/assets/58f01f2f-a61c-4199-a746-65b1c768797e)

**Comment by SupDos** [comment]
you can press ctrl + h to hide and unhide it

**Reply by Parsa-GP** [reply]
i know, but i want to step forward one frame and see the key is pressed or not.
in that case, i have to press ctrl+h or unhover my mouse everytime i want to check the key press state on that frame
and i know i can move down the keypress element in skin, but there's an element on the bottom of it and on top, it's my score and stuff.
It should be no problem in bigger screens but i have smol monitor (1366x768) and i can't scale the UI down bc of that.

**Reply by SupDos** [reply]
You could always just add a second keypress counter while you're doing this, then remove it once you're done, but fair point

**Reply by peppy** [reply]
There are hotkeys to step forward/backwards (<kbd>,</kbd> and <kbd>.</kbd> by default but can be rebound).

That said, I wouldn't be against moving these playback controls to a screen centre-bottom overlay in line with how a lot of media players do it.

**Reply by Parsa-GP** [reply]
I think with the introduction of analysis settings, there's no need to change the background opacity anymore.
